boardStupid.php

```diff
@@ -0,0 +1,24 @@
+<?php
+session_start();
+
+$db = new PDO('mysql:host=db; dbname=boardStupid', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT *  FROM `boardStupid`;');
+$query->execute();
+$result = $query->fetchAll();
+
+require_once 'game_function.php'
+
+?>
+
+  
+
+<?php
+
+echo '<h1>boardStupid</h1>';
```

Comment:
I would suggest removing this echo and just putting the HTML above the PHP tag on line 17.